### PR TITLE
Updates from master zoom ips

### PR DESCRIPTION
**Pull request by teizenman**

### CODE REVIEW

**Comment by ronykoz** [comment]
@teizenman you should update your base to the contrib/...

**Comment by teizenman** [comment]
@ronykoz Done

File: Packs/BPA/TestPlaybooks/playbook-BPA-test.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 id: Test-BPA
-version: -1
+version: 8
```
Comment (ronykoz) [inline Packs/BPA/TestPlaybooks/playbook-BPA-test.yml:2]: ?

Comment (teizenman) [reply Packs/BPA/TestPlaybooks/playbook-BPA-test.yml:2]: Its from master, Ill update it

**Pull request merged**